test/TestHelper.cpp

```diff
@@ -180,8 +236,16 @@ void ImportTest::exportTest(bytes const& _output, State const& _statePost)
 	// export logs
 	m_TestObject["logs"] = exportLog(_statePost.pending().size() ? _statePost.log(0) : LogEntries());
 
-	// export post state
+	// compare expected state with post state
+	if (m_TestObject.count("expect") > 0)
```

Comment:
that is how it is right now. if something is set in expected section, than it would be checked.
so you could check only sender account, or only coinbase account for certain state in the post state.